decidim-admin/app/views/decidim/admin/features/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<section id="features">
+  <div class="row">
+    <div class="columns">
+      <h3><%= t("features.title", scope: "decidim.admin") %></h3>
+    </div>
+    <div class="small-4 columns text-right">
+      <button class="dropdown button" data-toggle="add-feature-dropdown"><%= t "features.index.add", scope: "decidim.admin" %></button>
+      <div class="dropdown-pane" id="add-feature-dropdown" data-dropdown data-auto-focus="true" data-close-on-click="true">
+        <ul class="vertical menu add-features">
+          <% @manifests.each do |manifest| %>
+            <li><%= link_to t("#{manifest.name}.name", scope: "features"), new_participatory_process_feature_path(type: manifest.name), class: manifest.name %></li>
```

Comment:
Because the `name` of the manifest translates to the *kind of* component you're trying to create. Passing `name` could be confusing, as it isn't the name of the component itself.